pysd/py_backend/xmile/SMILE2Py.py

```diff
@@ -175,6 +192,13 @@ def visit_identifier(self, n, vc):
     def visit_func(self, n, vc):
         return functions[n.text.lower()]
 
+    def visit_build_call(self, n, vc):
+        builder_name = vc[0].lower();
+        arguments = [e.strip() for e in vc[4].split(",")]
+        name, structure = builders[builder_name](self.element, self.subscript_dict, arguments)
```

Comment:
This is great, and how we should probably do it in vensim2py

Reply:
Ok, I try to change way to call builders in vensim2py